admin/test/features/step_definitions/pupil_not_taking_check.rb

```diff
@@ -66,6 +66,7 @@
 Given(/^I am on the pupil reason page$/) do
   step 'I am on the pupil not taking check page'
   step 'I want to add a reason'
+  @page = pupil_reason_page
```

Comment:
Why do we need an instance variable here? @sinhaamit1980

Reply:
We need this for sticky banner steps which is common across Pupil check and Generate pin page